.pipelines/singletenancy/aks/e2e-step-template.yaml

```diff
@@ -1,14 +1,8 @@
 parameters:
   name: ""
-  clusterType: ""
   clusterName: ""
-  nodeCount: ""
```

Comment:
Since we removed these from step template, we should remove passing these attributes from jon template too ?